packages/backend/src/accounting/errors.ts

```diff
@@ -49,6 +40,33 @@ export enum TransferError {
 export const isTransferError = (o: any): o is TransferError =>
   Object.values(TransferError).includes(o)
 
+export function areAllAccountExistsErrors(
+  errors: CreateAccountErrorCode[]
+): boolean {
+  return areAllOfTypeAccountErrors(errors, [
+    CreateAccountErrorCode.exists_with_different_flags,
+    CreateAccountErrorCode.exists_with_different_user_data,
+    CreateAccountErrorCode.exists_with_different_ledger,
+    CreateAccountErrorCode.exists_with_different_code,
+    CreateAccountErrorCode.exists_with_different_debits_pending,
+    CreateAccountErrorCode.exists_with_different_debits_posted,
+    CreateAccountErrorCode.exists_with_different_credits_pending,
+    CreateAccountErrorCode.exists_with_different_credits_posted,
+    CreateAccountErrorCode.exists
+  ])
+}
+
+export function areAllOfTypeAccountErrors(
+  errorsOccurred: CreateAccountErrorCode[],
+  errToVerify: CreateAccountErrorCode[]
+): boolean {
+  for (const occurred of errorsOccurred) {
+    if (errToVerify.includes(occurred)) continue
+    return false
+  }
+  return true
```

Comment:
Could be simplified to:

```suggestion
  for (const occurred of errorsOccurred) {
    if (!errToVerify.includes(occurred)) return false
  }
  return true
```